### PR TITLE
making num_classes for inception model parameterized

### PR DESCRIPTION
**Pull request by singlas**

### CODE REVIEW

**Comment by fchollet** [comment]
The Keras abbreviation for 'number' is 'nb' throughout the codebase.

If you make this sort of change, you would have to make it across all vision models we have in `applications` (in order to keep the signatures consistent across models), and update the docs to reflect it. If you care about this change, then please do it. Otherwise, please close the PR.

**Comment by singlas** [comment]
@fchollet I have done the changes to all the models and updated the docs.

File: docs/templates/applications.md [hunk 1]
```diff
@@ -206,6 +206,7 @@ The default input size for this model is 299x299.
     It should have exactly 3 inputs channels,
     and width and height should be no smaller than 71.
     E.g. `(150, 150, 3)` would be one valid value.
+- nb_classes: optional number of categories to classify images into, only to be specified if `include_top` is True
```
Comment (fchollet) [inline docs/templates/applications.md]: - please rename to `classes` everwhere
- add: `and if no `weights` argument is specified.`.
- break the doc line to respect line length limitations.

Comment (singlas) [inline docs/templates/applications.md]: thanks for the review. 
did you mean rename `nb_classes` to `classes` or `categories` to `classes` ?
I will assume the latter and resubmit the PR. Let me know if you meant something else. @fchollet

**Comment by singlas** [comment]
thanks for the review.
did you mean rename `nb_classes` to `classes` or `categories` to classes ?
I have assumed the latter and done the changes. Let me know if you meant something else. @fchollet

**Comment by fchollet** [comment]
I actually meant the former. `classes` should be cleaner than `nb_classes`.

**Comment by singlas** [comment]
@fchollet fixed

**Comment by fchollet** [comment]
LGTM, thanks!

**Pull request merged**